pom.xml

```diff
@@ -67,6 +68,21 @@
             <artifactId>selenium-support</artifactId>
             <version>${selenium.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
```

Comment:
Is denpendency from line 71 to 75 is duplicated with line 51 to 55 anh @khiemkao

Reply:
removed dup dependencies